api_v3/lib/types/entry/KalturaLiveEntry.php

```diff
@@ -36,10 +36,17 @@
 	/**
 	 * Elapsed recording time (in msec) up to the point where the live stream was last stopped (unpublished).
 	 * @var int
+	 * @deprecated
 	 */
 	public $lastElapsedRecordingTime;
 
 	/**
+	 * Elapsed live streaming time (in msec) up to the point where the live stream was last stopped (unpublished).
+	 * @var int
+	 */
+	public $lastElapsedLiveDuration;
```

Comment:
lastElapsedLiveDuration replaces lastElapsedRecordingTime, which is set on the server in KalturaLiveEntryService::appendRecording(). 
This var is the live stream duration anchor, which is set upon a stop-stream (unpublish) event and will not change during a streaming session even when appendRecording is called periodically. This is opposed to LiveEntry::msDuration, which is changed each call to appendRecording via setLengthInMsec.